docs/examples/example_mitgcm.py

```diff
@@ -41,7 +41,6 @@ def periodicBC(particle, fieldset, time):  # pragma: no cover
         str(path),
         pset,
         outputdt=timedelta(days=1),
-        chunks=(len(pset), 1),
```

Comment:
I assume this was unintentional? Chunking on the particlefile would still be needed.

Reply:
Yep good point; that can be reverted!

Reply:
bfb1f62